### PR TITLE
Set up project with django-plotly-dash

### LINKED ISSUE #135: Set up repo to work with django-plotly-dash
Adding dash apps to django apps using `django-plotly-dash` requires a few global changes to the django settings, and a specific folder structure. The first step is to make these changes and test it out on a simple minimal dash app. This will allow #121 and #120 to proceed.

### PR DESCRIPTION
**Pull request by tsmbland**
This makes a few changes to the repo to get it set up with django-plotly-dash

I've created a minimal dash app accessible at http://localhost:8000/validated/daily_validation_dev/ which for now just consists of a single graph, just to see how everything is done. This will need to be added to and eventually integrated into the main page (the dev url/view should then be deleted). I am going to use this as a starting point for #121, and a similar approach could be used for #120

(I've also used github copilot to add docstrings and comments to the javascript code to make it easier for me to understand. However in the long run we are planning to get rid of this code anyway.)

Close #135

### CODE REVIEW

**Comment by tsmbland** [comment]
Not sure why the tests are failing? @dalonsoa

**Review by CWestICL — APPROVED** [review]
Other than the failing tests, all looks good to me! Tested on my PC and it works as expected.

Not sure why the tests are failing either, looks like a migration issue but you haven't changed any of the models. Hopefully @dalonsoa has some insight

File: validated/urls.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 from rest_framework.urlpatterns import format_suffix_patterns
 
 from . import views
+from .dash_apps.finished_apps import daily_validation
```
Comment (dalonsoa) [inline validated/urls.py:18]: Why do you import `daily_validation` here?

**Review by dalonsoa — COMMENTED** [review]
OK, the problem is that when launching the app for the first time, there's no data in the database. As the `daily_validation` code is executed when the application is launch for the first time rather than when the page is loaded, there's not time to populate the db, and things crash. This is not going to be a problem in the real scenario, obviously, but it is now during development.

For you, this has worked because you already had the database populated, but in the GH Actions or in my laptop, in a totally fresh installation, it breaks. 

Would my suggestion work, loading the problematic code only when the relevant view is loaded?

File: validated/views.py [hunk 2]
```diff
@@ -426,6 +427,11 @@ def post(self, request, *args, **kwargs):
         return render(request, "home/message.html", {"message": form})
 
 
+class DailyValidationDev(View):
+    def get(self, request, *args, **kwargs):
+        return render(request, "daily_validation_dev.html")
```
[the remark this answers is not in the document]
Comment (tsmbland) [reply validated/views.py:434]: This worked! Thanks

**Pull request merged**